linter.py

```diff
@@ -87,8 +95,11 @@ def split_match(self, match):
 
                 line = max(int(match.group('line')) - 1, 0)
                 col = int(match.group('col')) - 1
+                another_line = int(match.group('another_line') or line)
+                col_end = int(match.group('col_end'))
+                near_length = self.view.text_point(another_line, col_end) - self.view.text_point(line, col)
 
                 # match, line, col, error, warning, message, near
-                return match, line, col, True, False, message, None
+                return match, line, col, True, False, message, " " * near_length
```

Comment:
On second thought, if you wanted to actually grab the text from the buffer between the start and end col near would become useful.